src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = Object.assign({}, state);
```

Comment:
```suggestion
  let stateCopy = Object.assign({}, state);
```

Reply:
It is not entirely clear why this is necessary